### ISSUE TITLE
Unable to execute provider in alpine

### ISSUE THREAD

**Issue by rickardrosen** [terraform-lxd/terraform-provider-lxd#125]
Hi,
We use an alpine docker container to get a standardized terraform environment but cannot use this provider.
I'm also unable to compile it as there seems to be an integration issue with the current LXD go lib:

```
../../lxc/lxd/shared/archive_linux.go:79:15: undefined: DeviceTotalMemory
../../lxc/lxd/shared/network_linux.go:20:9: undefined: ExecReaderToChannel
```

Would it be possible to request release a build with CGO_ENABLED=0?

Thanks!

Rickard

**Comment by sl1pm4t** [comment]
Hi @rickardrosen 
I'm happy to add `CGO_ENABLED=0` to the build - have you been able to confirm it fixes your issue? Or are you still unable to build due to the LXD go lib error?
Matt

**Comment by rickardrosen** [comment]
Last I tried it failed.
Haven’t dug into it other than that my own provider for our
configdb/inventory required the cgo flag disabled for it to run under
alpine.

If you build it I’ll be happy to test the binary.

Regards
Rickard

**Comment by adamcstephens** [comment]
This was fixed by #205 

https://github.com/terraform-lxd/terraform-provider-lxd/blob/ce56b737fd230034b27c9b9aa4bdd09da8298901/.goreleaser.yml#L12
